addons/csw/stringtable.xml

```diff
@@ -422,6 +455,7 @@
         </Key>
         <Key ID="STR_ACE_CSW_GMGBelt_displayName">
             <English>[CSW] 20mm Grenade GMG Belt</English>
```

Comment:
@TheCandianVendingMachine: Grenade Grenade Machine Gun?

@padddyyy: Vanilla-Strings vergleichen, eventuell GraMaWa anstelle der englischen Abkürzung.

Reply:
I think the idea was 20mm Grenade, [for a] GMG belt. But yeah you make sense. All messed up strings were probably written by sleep deprived me at 5 am :P